index.bs

```diff
@@ -170,2193 +138,170 @@ The important commonality between them being that they offer some degree of spat
 
-The terms [=3DoF=] and [=6DoF=] are used throughout this document to describe the tracking capabilities of [=/XR devices=].
-
- - A <dfn>3DoF</dfn> device, short for "Three Degrees of Freedom", is one that can only track rotational movement. This is common in devices which rely exclusively on accelerometer and gyroscope readings to provide tracking. [=3DoF=] devices do not respond translational movements from the user, though they may employ algorithms to estimate translational changes based on modeling of the neck or arms.
- - A <dfn>6DoF</dfn> device, short for "Six Degrees of Freedom", is one that can track both rotation and translation, enabling for precise 1:1 tracking in space. This typically requires some level of understanding of the user's environment. That environmental understanding may be achieved via inside-out tracking, where sensors on the tracked device itself (such as cameras or depth sensors) are used to determine the device's position, or outside-in tracking, where external devices placed in the user's environment (like a camera or light emitting device) provides a stable point of reference against which the [=/XR device=] can determine it's position.
-
-Application flow {#applicationflow}
-----------------
-
-<section class="non-normative">
-
-Most applications using the WebXR Device API will follow a similar usage pattern:
-
-  * Query {{XR/supportsSession()|navigator.xr.supportsSession()}} to determine if the desired type of XR content is supported by the hardware and UA.
-  * If so, advertise the XR content to the user.
-  * Wait for the user to [=triggered by user activation|trigger a user activation event=] indicating they want to begin viewing XR content.
-  * Request an {{XRSession}} within the user activation event with {{XR/requestSession()|navigator.xr.requestSession()}}.
-  * If the {{XRSession}} request succeeds, use it to run a [[#frame|frame loop]] to respond to XR input and produce images to display on the [=XRSession/XR device=] in response.
-  * Continue running the [[#frame|frame loop]] until the [=shut down the session|session is shut down=] by the UA or the user indicates they want to exit the XR content.
-
-</section>
-
-Model {#model}
-=====
-
-XR device {#xr-device-concept}
----------
-
-An <dfn for="">XR device</dfn> is a physical unit of hardware that can present imagery to the user. On desktop clients, this is usually a headset peripheral. On mobile clients, it may represent the mobile device itself in conjunction with a viewer harness. It may also represent devices without stereo-presentation capabilities but with more advanced tracking.
+XR Devices often have additional handheld hardware that can be spatially tracked and are used for input.  These devices are often referred to as "motion controllers", "handheld controllers", or "tracked controllers". Often motion controllers will have at least one button, trigger, thumbstick, or touchpad.
```

Comment:
Maybe strike the word "handheld" here? Don't want to exclude something like the GearVR touchpad, which should be able to expose gamepad inputs but won't be a `tracked-pointer` device or use the `xr-standard` mapping.

Reply:
Totally get that, and I do call that out lower down in the document.  If I remove "handheld" here, does that make the rest of the paragraph nonsensical?

Reply:
Oh, yeah. On re-reading this probably needs larger changes. Maybe something like this?

```
XR Devices often have additional controller hardware that allows users to interact with immersive experiences with button, trigger, thumbstick, or touchpad inputs. Frequently these devices are spatially tracked as well, and referred to as "motion controllers", "handheld controllers", or "tracked controllers".
```